src/mbgl/util/chrono.cpp

```diff
@@ -3,6 +3,13 @@
 #include <parsedate/parsedate.h>
 
 #include <cstdio>
+#include <ctime>
+
+#if defined(_WINDOWS)
+#define _gmtime(t, i) gmtime_s(i, t)
+#else
+#define _gmtime(t, i) gmtime_r(t, i)
```

Comment:
[`gmtime_s`](http://en.cppreference.com/w/c/chrono/gmtime) is standard since C++11 if you define the right preprocessor symbols. Can we do that instead?